### PR TITLE
Add calculater app "Add Hours and Minutes"

### PR DESCRIPTION
**Pull request by kebekus**
Please consider my app for publication on flathub.org

### CODE REVIEW

**Comment by scx** [comment]
It looks ok, but you might consider using YAML instead of JSON.
```
app-id: de.akaflieg_freiburg.cavok.add_hours_and_minutes
runtime: org.kde.Platform
runtime-version: '5.11'
sdk: org.kde.Sdk
command: addHoursAndMinutes
finish-args:
  - --share=ipc
  - --socket=x11
  - --socket=wayland
  - --device=dri
modules:
  - name: addHoursAndMinutes
    buildsystem: cmake-ninja
    config-opts:
      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
    sources:
    - type: git
      url: https://gitlab.com/kebekus/addHoursAndMinutes.git
      tag: 0.9.3
      commit: 1a64502054d6e7983e9880a5e4c73291ef5dcc75
```

See also: https://blogs.gnome.org/alexl/2018/05/04/yamling-the-flathub/

**Comment by barthalion** [comment]
It's personal preference, not actual recommendation by Flathub team.

bot, build de.akaflieg_freiburg.cavok.add_hours_and_minutes

File: de.akaflieg_freiburg.cavok.add_hours_and_minutes.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "app-id": "de.akaflieg_freiburg.cavok.add_hours_and_minutes",
```
Comment (barthalion) [inline de.akaflieg_freiburg.cavok.add_hours_and_minutes.json]: Do you own this domain exist? If not, `com.gitlab.kebekus.addHoursAndMinutes` would be a better choice.

File: de.akaflieg_freiburg.cavok.add_hours_and_minutes.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "addHoursAndMinutes",
```
Comment (barthalion) [inline de.akaflieg_freiburg.cavok.add_hours_and_minutes.json]: `addhoursandminutes`

Comment (kebekus) [reply de.akaflieg_freiburg.cavok.add_hours_and_minutes.json]: Thank you for spotting the issue! I fixed that.  -- And by the way, thanks for your awesome work (on a Saturday night!)

**Comment by nedrichards** [comment]
bot, build de.akaflieg_freiburg.cavok.add_hours_and_minutes

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/de.akaflieg_freiburg.cavok.add_hours_and_minutes

Thanks!

**Pull request closed without merging**